add taskflow api example for worker queue assignment (#2098)
astronomer/docs: astro/configure-worker-queues.md

By default, all tasks run in the default worker queue. To run tasks on a different worker queue, assign the task to the worker queue in your DAG code.

> ### Step 1: Copy the name of the Worker Queue you want to assign tasks to.

Reviewer: ```suggestion
### Step 1: Copy the name of the worker queue
```

1. In the Cloud UI, select a Workspace and select a Deployment.

2. Click the **Worker Queues** tab.

3. Copy the name of the worker queue name you want to assign a task to.

> ### Step 2: Change the DAG Code to assign the Airflow tasks to a specific Worker Queue:

Reviewer: ```suggestion
### Step 2: Assign the task in your DAG code
```
